README.md

```diff
@@ -27,101 +27,26 @@ However, if you want to use it alongside AdGuard browser extension or any other
 
 Alternatively, you can install the beta version of AdGuard Extra userscript: https://userscripts.adtidy.org/beta/adguard-extra/1.0/adguard-extra.user.js
 
+## How to exclude websites
+
+### Userscript
+
+You can restrict userscript from working on certain websites by adding `@exclude` rule to the userscript header. For example, to exclude `example.com`, you should add the following line to the userscript header:
```

Comment:
Probably worth to write how to allow extensions to work on certain sites.
https://www.tampermonkey.net/documentation.php?locale=en#meta:include

Reply:
in case user is paranoid)